mts-config-server/pom.xml

```diff
@@ -1,23 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
not sure the new name is easier to understand. IMO sc-xxx means this is part if the spring cloud set of microservices (together with discovery and gateway). wdyt

Reply:
The name should be MTS-xx as it is a server that servers MTS config files not spring files. We shouldnt have references to the 3rd party libraries that we use to help build/wire the app....

Reply:
as I see it, its not about what it serves. its about what kind of service this is.
and in this case its not our 'custom application' it is a supportive piece of service. which in this case is a spring cloud type.
everything in this project is MTS. so it doesnt add info. 
you can look at it the same way as the arch diagram, in it, the micro services are grouped to 'custom' apps and sc apps.

Reply:
sorry but I still disagree :) - this is an MTS server, sure it uses spring but so do all the classes and we dont name them springcontroller etc - just like you wouldnt name domain classes that rely on Hibernate hibernatePersonxx etc etc. If we had more than 1 project that used a config server which relied on spring would all the config servers be named the same? I dont think they should..they should be named for their purpose which is to server project related files etc

Reply:
talked offline. we'll remove the prefix